### DISCUSSION TITLE
Feature Request: show mutations in @tanstack/react-query-devtools

### DISCUSSION THREAD

**Discussion by zzzev** [TanStack/query#4590] [Ideas]
It would be great to be able to browse the list of mutations, see their status, inputs, outputs, etc. in a similar way to how the queries are visualized now.

Would love to hear thoughts/feedback from maintainers and other users.

**Comment by josephmarkus** [comment]
Would be great to have this aspect documented, even if to say "You can use these devtools to observe queries, but not mutations": https://github.com/TanStack/query/blob/main/docs/react/devtools.md

Proposed PR for the above: https://github.com/TanStack/query/pull/4810
